### DISCUSSION TITLE
CSV Wide date format ISO 8601 in the validator

### DISCUSSION THREAD

**Discussion by sklukasatphs** [CMSgov/hospital-price-transparency#104] [Q&A] [answered]
As other users have mentioned the date format YYYY-MM-DD when the .csv opens out of Excel the date automatically goes to MM-DD-YYYY.  we tried adding apostrophes to keep the format, ran it through the validator and received the following error:

"last_updated_on" value "2024-07-01" is not a valid format. You must encode the date using the ISO 8601 format: YYYY-MM-DD or the month/day/year format: MM/DD/YYYY, M/D/YYYY

is there an update to the regulation that approves of either MM/DD/YYYY, M/D/YYYY formats?  Based on the validator results we cannot assume that a non ISO 8601 format is acceptable.

**Comment by RidenourK** [comment, marked as answer]
@sklukasatphs Thank you for your question. Please see the response to GitHub discussion post #105.
